lib/Site.js

```diff
@@ -443,7 +444,8 @@ Site.prototype.buildSourceFiles = function () {
 };
 
 Site.prototype._rebuildAffectedSourceFiles = function (filePaths) {
-  const uniquePaths = _.uniq(filePaths);
+  const wrappedFilepaths = Array.isArray(filePaths) ? filePaths : [filePaths];
```

Comment:
- `wrapped` seems like an odd way to describe an "ensuring an array", but I don't have a better name at the moment. 
- We should not lose the capital `P` for consistency.

Reply:
How about just filePathArray?